### DISCUSSION TITLE
纯CPU构建知识库问题

### DISCUSSION THREAD

**Discussion by kexuezhilu** [wenda-LLM/wenda#270] [General]
兄弟们好，![微信截图_20230511104141](https://github.com/l15y/wenda/assets/125812696/9726b3c6-a421-406c-8476-f5ddcda7d55c)
 纯CPU  ，16G，构建知识库，截取三国演义中的部分片断，片段中包含刘备两个字，但是打开知识库，去测试报错如下，该如何处理呢？

**Comment by l15y** [comment]
上面肯定还有报错
